ci-operator/config/openshift/sre-ssh-proxy-container/openshift-sre-ssh-proxy-container-master.yaml

```diff
@@ -0,0 +1,32 @@
+base_images:
+  os:
+    name: centos
+    namespace: openshift
+    tag: "7"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+promotion:
+  name: "4.6"
+  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.6"
+  namespace: ocp
+tests:
+- as: make
+  commands: make
+  container:
+    from: src
```

Comment:
We don't have access to the raw daemon socket, so we need to use image stream builds in CI
```suggestion
```